cobalt/BUILD.gn

```diff
@@ -66,22 +66,26 @@ group("default") {
     "//third_party/brotli:dec_no_dictionary_data",
   ]
 
-  if (sb_is_evergreen) {
+  if (sb_is_modular) {
     deps += [
       "//base/util/values:values_util",
-      "//cobalt/updater",
-      "//cobalt/updater:crash_sandbox",
-      "//cobalt/updater:noop_sandbox",
-      "//cobalt/updater:one_app_only_sandbox",
-      "//components/client_update_protocol",
-      "//components/crx_file",
-      "//components/prefs",
-      "//components/update_client",
       "//third_party/llvm-project/compiler-rt:compiler_rt",
```

Comment:
We probably don't need those dependencies here at all as add them whenever you depend on the starboard platform. You can keep them for now, but we should try to clean it up in separate change.